monkey/common/agent_configuration/agent_sub_configurations.py

```diff
@@ -177,7 +181,7 @@ class ExploitationOptionsConfiguration(MutableInfectionMonkeyBaseModel):
     """
 
     http_ports: Tuple[NetworkPort, ...] = Field(
-        title="HTTP Ports",
+        title="HTTP ports",
         description="List of ports the monkey will check if are being used for HTTP",
```

Comment:
Suggested change: 
A list of ports that the Monkey checks for HTTP traffic
@VakarisZ let me know if "HTTP traffic" is the right term here

Reply:
Monkey doesn't check for traffic on these ports. It just tries to figure out whether these ports are serving HTTP

Reply:
I've done "List of ports the Agent will check for using HTTP protocol", but this option will be short lived

Reply:
"A list of ports the Monkey checks for HTTP services"
Is that better or am I still not nailing it?

Reply:
Why short lived? Will this change any time soon?

Reply:
Final: 
List of ports the Agent will check for using an HTTP protocol